doc/rules/no-equal-arguments.md

```diff
@@ -1,5 +1,7 @@
 # No equal arguments
 
+![Configurable](https://img.shields.io/badge/-configurable-informational)
```

Comment:
@incendial please add example of configuration

Reply:
Done.